### PR TITLE
docs: update cli

### PR DESCRIPTION
**Pull request by wizardofhogwarts**

### CODE REVIEW

File: src/content/api/cli.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ webpack --json
 webpack --json > stats.json
 ```
 
-In every other case, webpack prints out a set of stats showing bundle, chunk and timing details. Using this option, the output can be a JSON object. This response is accepted by webpack's [analyse tool](https://webpack.github.io/analyse/), or chrisbateman's [webpack-visualizer](https://chrisbateman.github.io/webpack-visualizer/), or th0r's [webpack-bundle-analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer). The analyse tool will take in the JSON and provide all the details of the build in graphical form.
+In every other case, webpack prints out a set of stats showing bundle, chunk and timing details. Using this option, the output can be a JSON object. This response is accepted by webpack's [analyse tool](https://webpack.github.io/analyse/), or chrisbateman's [webpack-visualizer](https://chrisbateman.github.io/webpack-visualizer/), or th0r's [webpack-bundle-analyzer](https://github.com/webpack-contrib/webpack-bundle-analyzer). The analysis tool will take in the JSON and provide all the details of the built-in graphical form.
```
Comment (jamesgeorge007) [inline src/content/api/cli.md]:
```md
The analysis tool will take in the JSON and provide all the details of the built-in graphical form.
```

The former version seems fine. Please revert this change.

Comment (jamesgeorge007) [inline src/content/api/cli.md]: The former version seems fine. Please revert this change.

Comment (wizardofhogwarts) [reply src/content/api/cli.md]: Yes I also agree after reading again. Thanks for feedback.

**Comment by EugeneHlushko** [comment]
Thanks!
